deploy/roles/elasticsearch/tasks/main.yml

```diff
@@ -45,15 +45,15 @@
 
 - name: Open ports for Elasticsearch transport communication
   command: "iptables -I {{ iptables_chain }} 3 -m state --state NEW -p tcp --dport {{ es_transport_ports }} -j ACCEPT"
-  when: iptables_config
+  when: iptables_config and proxy_server_deployed
```

Comment:
@stirlingv We don't need to add the variable here, since this command doesn't rely on the proxy server.